### PR TITLE
(DOCSP-10023): IoT SDK installation instructions

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
Stage: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/DOCSP-10023/get-started/install-node.html

JIRA: https://jira.mongodb.org/browse/DOCSP-10023

### CODE REVIEW

File: source/get-started/install-node.txt [hunk 1]
```diff
@@ -27,6 +27,13 @@ application development.
 - For cross-platform mobile app development using web frameworks, use the {+service+} :doc:`React Native SDK </get-started/install-react-native>`.
 - {+service+} does not officially support other front-end frameworks, such as `Electron <https://electronjs.org/>`__.
 
+.. note::
+
+   For `Raspberry Pi <https://www.raspberrypi.org/>`__ 2, 3, 4, and
+   Compute devices, you can install the IoT (Internet of Things) release
```
Comment (cbush) [inline source/get-started/install-node.txt:33]: How?

Comment (nathan-contino-mongo) [reply source/get-started/install-node.txt:33]: The installation works the exact same way (through npm) as the x86 node SDK. I don't think we need to confuse users with two identical sets of instructions

Comment (cbush) [reply source/get-started/install-node.txt:33]: Something in the package.json for the Raspberry Pi project I guess informs npm to build for ARM?

Comment (mongomoe) [reply source/get-started/install-node.txt:33]: will this string "install the IoT (Internet of Things) release" eventually link to a doc with install instructions?

**Comment by nathan-contino-mongo** [comment]
Closing because IDK what's going on with the IoT SDK

**Pull request closed without merging**